modules/integration/src/test/java/io/ballerina/messaging/broker/integration/util/ClusterUtils.java

```diff
@@ -0,0 +1,114 @@
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.messaging.broker.integration.util;
+
+import org.awaitility.Awaitility;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.Socket;
+import java.util.concurrent.TimeUnit;
+import javax.jms.ConnectionFactory;
+
+public class ClusterUtils {
+
+    private static final String GET_PID = "lsof -ti tcp:";
+    private static final String KILL_PID = "kill -9 ";
+    private static final String SHUTDOWN = "kill -s TERM ";
+    private static ConnectionFactory connFactory;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClusterUtils.class);
+
+    /**
+     * Method to kill broker node
+     *
+     * @param port port of the broker node to be killed
+     */
+    public static void killBrokerNode(String port) throws IOException {
+        Process process = Runtime.getRuntime().exec(GET_PID + port);
+        BufferedReader stdInput = new BufferedReader(new InputStreamReader(process.getInputStream()));
+        String pid = stdInput.readLine();
+        Runtime.getRuntime().exec(KILL_PID + pid);
+    }
+
+    /**
+     * Method to start broker node
+     *
+     * @param brokerHome path to the broker home to be started
+     */
+    public static void startBrokerNode(String brokerHome) throws IOException {
+        Runtime.getRuntime().exec("sh " + brokerHome + "/bin/broker");
+    }
+
+    /**
+     * Method to shutdown broker node
+     *
+     * @param port port of the broker home to be shutdown
+     */
+    public static void shutdownBrokerNode(String port) throws IOException {
+        Process process = Runtime.getRuntime().exec(GET_PID + port);
+        BufferedReader stdInput = new BufferedReader(new InputStreamReader(process.getInputStream()));
+        String pid = stdInput.readLine();
+        Runtime.getRuntime().exec(SHUTDOWN + pid);
+    }
+
+    /**
+     * Method to restart broker node
+     *
+     * @param brokerHome path to the broker home to be restarted
+     * @param portOne    port of the broker home to be restarted
+     */
+    public static void restartBrokerNode(String brokerHome, String portOne, String hostnameTwo, String portTwo)
+            throws IOException {
+        shutdownBrokerNode(portOne);
+        Awaitility.await().atMost(30, TimeUnit.SECONDS)
+                .pollInterval(2, TimeUnit.SECONDS)
+                .until(() -> !ClusterUtils.isPortAvailable(hostnameTwo, Integer.parseInt(portTwo)));
+        startBrokerNode(brokerHome);
+    }
+
+    /**
+     * Method for check port availability
```

Comment:
Following is a good read on how to write javadoc properly.
https://www.oracle.com/technetwork/articles/java/index-137868.html

Reply:
I will follow and update